### ISSUE TITLE
Examples for Fomu?

### ISSUE THREAD

**Issue by pdp7** [litex-hub/fpga_101#3]
@enjoy-digital @mithro do you think it is feasible to modify this tutorial to use the ice40 Fomu instead of Nexys4DDR?

**Comment by enjoy-digital** [comment]
@pdp7: i was also think about that: try to use an open-source board with open-source tools. The good candidate are Fomu, iCEBreaker, ULX3S or OrangeCrab. We would just need to see what would be the best. Fomu could be the best one yes, but that could be worth discussing a bit.

The reason we are using Nexys4DDR here is that all these open-source boards were not existing at the time it was created and the university where i was giving these lessons/labs had lots or Nexys4DDR :)

**Comment by mithro** [comment]
I think the iCEBreaker is probably the strongest candidate having the right set of peripherals?

**Comment by enjoy-digital** [comment]
I also really like the iCEBreaker for this since having an UART/JTAG chip on the board avoid having to integrate a CPU / USB Stack and creating very simple initial SoCs + it has more Leds/Buttons/GPIOs.

**Comment by pdp7** [comment]
Yes, iCEBreaker does seem like a good target for getting started tutorials.
